src/libNode/PoWProcessing.cpp

```diff
@@ -182,6 +182,10 @@ bool Node::StartPoW(const uint64_t& block_num, uint8_t ds_difficulty,
         LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
                   "Time out while waiting for DS Block");
 
+        if (!m_mediator.m_lookup->AlreadyJoinedNetwork()) {
```

Comment:
May be log a message like “I lost pow again. I was not in network for last DS epoch.”

Reply:
ok